pkg/packer/manifests/vsphere/packer.pkr.hcl

```diff
@@ -104,9 +104,9 @@ variable "guest_os_type" {
   type = string
 }
 
-variable "ib_version" {
+variable "kib_version" {
```

Comment:
we are not using `KIB_VERSION` at all. we should remove it and its references from this file.